src/components/modal/Modal.scss

```diff
@@ -133,3 +135,33 @@
     z-index: 89;
     display: flex;
 }
+
+//Responsive support
+.is-responsive-web {
+    .modal {
+        @include breakpoint($modal-fullscreen) {
+            padding: 0;
+        }
+    }
+
+    .modal-dialog-container {
+        width: 100%;
+        margin: auto;
+
+        @include breakpoint($modal-fullscreen) {
+            height: 100%;
+
+            .modal-dialog {
+                min-height: 100%;
+                padding: 22px 16px 12px 16px;
+                border-radius: 0;
+            }
+        }
+    }
+
+    .modal-close-button {
+        @include breakpoint($modal-fullscreen) {
+            right: 10px;
```

Comment:
Can we retain the existing 20px value, instead?

Reply:
This was also pulled from the Zeplin designs.